firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/Utils.java

```diff
@@ -57,25 +56,6 @@ public static <T> Task<T> race(Task<T> t1, Task<T> t2) {
     return result.getTask();
   }
 
-  /**
```

Comment:
Wonder of we can get rid of the race method above? Eventually feels like we can get rid of this whole file.

Reply:
Done! Yes, now it's just the await methods. We can reimplement them in the CrashlyticsTasks util and get rid of this whole class. It's too bad we can't use the Tasks await method because it specifically disallows the main thread.